gcloud/datastore/connection.py

```diff
@@ -317,9 +304,9 @@ def commit(self, project, request_pb):
         request_pb.project_id = project
         try:
             return self._stub.Commit(request_pb, GRPC_TIMEOUT_SECONDS)
```

Comment:
Also I notice `credentials` as a keyword arg, can we just use that or is it better to keep using `make_stub`?

```python
In [14]: stub.Commit?
Type:           _UnaryUnaryMultiCallable
String form:    <grpc._channel._UnaryUnaryMultiCallable object at 0x7ff96bfdb050>
File:           /usr/local/lib/python2.7/dist-packages/grpc/_channel.py
Signature:      stub.Commit(self, request, timeout=None, metadata=None, credentials=None)
Docstring:      <no docstring>
Call signature: stub.Commit(request, timeout=None, metadata=None, credentials=None)
```